src/plugins/intel_gpu/include/intel_gpu/runtime/layout.hpp

```diff
@@ -348,6 +367,8 @@ struct padding {
                            ///< to it using round-towards-nearest-even (for floating-point data types) or round-towards-zero (for integral
                            ///< data types).
 
+    tensor dynamic_pad_dims = tensor(0);
```

Comment:
As this dynamic_pad_dims variable is used only as a marker, doesn't make sense to use something more optimized (like vector/array of bool values or bitset)?

Reply:
Yeah it can be vector, bitset, someghing. As a initial impl, I maded it as tensor to sync with other dimension treating logics.